### ISSUE TITLE
gammastep not working on latest master

### ISSUE THREAD

**Issue by kevenwyld** [hyprwm/Hyprland#2512]
Launching gammastep on the latest build from github doesn't work at all.  Gammastep launches, but nothing happens, and no errors are reported.  It works fine on `v0.26.0`.

There aren't any debug logs in hyprland.log related to gammastep, only things like setting the window title of the terminal I was using.

Here's the output from gammastep, which seems to think it's working fine....

```
] > gammastep -vvv
Notice: Solar elevations: > 3.0 (Day), < -6.0 (Night)
Notice: Temperatures: 4800K (Day), 3000K (Night)
Notice: Brightness: 1.00:1.00
Notice: Gamma (Day): 1.000, 1.000, 1.000
Notice: Gamma (Night): 1.000, 1.000, 1.000
Informational: Using method: 'wayland'
Informational: Waiting for initial location to become available...
Notice: Location: <redacted>
Notice: Color temperature: 6500K
Notice: Brightness: 1.00
Notice: Status: Enabled
Notice: Period: Daytime
Notice: Color temperature: 4800K
Debug: Adjusting temperature
Debug: Adjusting temperature
Debug: Adjusting temperature
Debug: Adjusting temperature

^CNotice: Status: Disabled
Debug: Currently disabled
Notice: Period: None
```

I tried disabling VRR, and also reducing the number of displays as I have a somewhat complicated display arrangement but nothing seemed to make a difference.

I didn't see anywhere whether you officially support gammastep or not, so if you don't feel free to close this.

Thanks!

**Comment by ALizarazoTellez** [comment]
I've been looking for a solution to this problem for about an hour, I thought the problem was gammastep, but yes, I just confirmed, the problem is Hyprland, the workaround (in Arch Linux) is to use the `hyprland` package from the `extra` repository and not `hyprland-git` from AUR (if you don't want to compile things of course).

**Comment by vaxerski** [comment]
fixed in fbabb105c3427fbbce4dfc3ea9701ba473298206
